Library/Homebrew/livecheck/livecheck.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 require "livecheck/strategy"
+require "ruby-progressbar"
```

Comment:
Is this a gem that would be vendored? How many dependencies does it have? As @samford mentioned: does it support TTY handling? If not: we should handle this ourselves.

Reply:
We already have it vendored I think. Known for not having dependencies based on what I saw.

Reply:
> Known for not having dependencies based on what I saw.

Cool!



> We already have it vendored I think.

It's vendored but `gitignored` so https://github.com/Homebrew/brew/blob/master/.gitignore#L133 would need removed.



> As for TTY handling, I erase the progress bar using our own `Tty` module.

I was thinking it should be disabled if it's not a TTY e.g. `unless $stdout.tty?`.

Reply:
Alright, I'll go ahead and just not have the progress bar unless it's a TTY 👍🏼